src/main/java/org/terasology/climateConditions/BodyTemperatureChangedEvent.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.climateConditions;
+
+import org.terasology.entitySystem.event.Event;
+
+public class BodyTemperatureChangedEvent implements Event {
+    public enum BodyTemperatureLevel {
+        CRITICAL_LOW, LOW, REDUCED, NORMAL, RAISED, HIGH, CRITICAL_HIGH;
+    }
+
+    private BodyTemperatureLevel bodyTemperatureLevel;
```

Comment:
I'm honestly not sure what a better practice is here. As this can only be set in the constructor, and is only read, we may as well make this a 

```suggestion
    public final BodyTemperatureLevel bodyTemperatureLevel;
```

and don't provide the getter method. I don't know what's better, though 🤷‍♂️

Reply:
In addition, the event should contain both values for _before_ (old) and _after_ (new).